src/components/formatters/src/generic_json_formatter.cc

```diff
@@ -48,8 +49,8 @@ void GenericJsonFormatter::ToString(const ns_smart_objects::SmartObject& obj,
 bool GenericJsonFormatter::FromString(const std::string& str,
                                       ns_smart_objects::SmartObject& out) {
   Json::Value json_root;
-  Json::Reader reader;
-  bool result = reader.parse(str, json_root);
+  JsonReader reader;
+  bool result = reader.parse(str, &json_root);
```

Comment:
const?

Reply:
@AByzhynar done in [1345b2e](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/1345b2e4f52415e896a5fa3d232aa391c69751bb)